backend/src/main/resources/application-dev.properties

```diff
@@ -20,7 +20,7 @@ thread.name.prefix=default_executor_thread
 spring.cloud.gcp.sql.enabled=false
 spring.datasource.url=jdbc:mysql://localhost:3306/rolecall_db?serverTimezone=UTC
 spring.datasource.username=rolecall
-spring.datasource.password=1234
+spring.datasource.password=Abcd1234@@
 
```

Comment:
I set my system up with the 1234 password. I don't want to change.

Reply:
fixed